src/content/news.js

```diff
@@ -4,70 +4,64 @@ import PrevNextButton from './prev_next'
-            },
-            {
-                id: 4,
-                src: "null",
-                alt: "",
-                title: "Warren Buffett compares the Depression to current day economy - Yahoo Finance",
-                description: "Warren Buffett compares the Depression to current day economyYahoo Finance Bill Murray asked Warren Buffett a question, and the billionaire gave a powerful answer on income inequalityYahoo Finance Warren Buffett Expects Market Doom That Can Take Down Bitcoin …",
-                url: "https://slashdot.org/firehose.pl?op=view&amp;id=130510362",
-                content: "The Fine Print: The following comments are owned by whoever posted them.We are not responsible for them in any way."
-            },
-            {
-                id: 5,
-                src: "https://dailyfintech.com/wp-content/uploads/2020/05/christiann-koepke-0jPuWm8_9wY-unsplash-1024x683.jpg",
-                alt: "Picture",
-                title: "Trading Through Turbulent Times: Opportunities For Payments’ Fintechs & Investors",
-                description: "Jessica Ellerm is a thought leader specializing in Small Business and the Gig Economy and is the CEO and Co- Founder of Zuper, a neowealth disruptor in Australia For many fintech companies and new financial technologies(hello Bitcoin), COVID - 19 is proving to …",
-                url: "https://dailyfintech.com/2020/05/06/trading-through-turbulent-times-opportunities-for-payments-fintechs-investors/",
-                content: "Jessica Ellerm is a thought leader specializing in Small Business and the Gig Economy and is the CEO and Co-Founder of Zuper, a neowealth disruptor in Australia\r\nFor many fintech companies and new financial technologies (hello Bitcoin), COVID-19 is proving to… [+2649 chars]"
-            }
-            ]
+            i: 0,
+            news: []
         }
+
+    }
+    handleLast = () => {
+        console.log("Last");
+        this.setState((state) => ({
+            i: (state.i - 5 >= 0) ? state.i - 5 : state.i
+        }))
+        this.arrivalHandle(this.url);
+
+    }
+    handleNext = () => {
+        console.log("Next");
+        this.setState((state) => ({
+            i: (state.i + 5 <= state.news.length) ? state.i + 5 : state.i
+            // i: state.i + 5
+        }))
+        this.arrivalHandle(this.url);
+    }
+
+    componentDidMount() {
+        this.arrivalHandle(this.url);
+    }
+
+    arrivalHandle = (url) => {
+        console.log(this.state)
+        fetch(url).then(response => (
+            response.json()
+        )).then(body => {
+            this.setState(state => ({
+                news: body.articles.splice(state.i, state.i + 5)
```

Comment:
if you already fetched all news, why not save all of them, but just don't show all of them? Also check if your API supports pagination?